src/lsg/patterns/link/demo.tsx

```diff
@@ -3,36 +3,22 @@ import Link from './index';
 import * as React from 'react';
 import Space, { Size } from '../space';
 
-const clickHandler = (event: React.MouseEvent<HTMLElement>): void => console.log(event);
+const clickHandler = (event: React.MouseEvent<HTMLElement>): void => {}; //console.log(event);
```

Comment:
In the demo.tsx files it should be ok to have `console.log`s.
These files are only used when working on the livingstyleguide. This was described wrong in the task.

Reply:
Ok.